frontend/src/components/grantaccess/form/UserNameInput.vue

```diff
@@ -67,12 +67,16 @@ const verifyUserId = async () => {
                 FamLoginUserState.state.value.famLoginUser!.organization
             }`;
         }
+    } finally {
+        emit(
+            'setVerifyResult',
+            verifiedUserIdentity.value?.found,
+            verifiedUserIdentity.value?.guid
+        );
     }
-    if (verifiedUserIdentity.value?.found)
-        emit('setVerifyResult', true, verifiedUserIdentity.value.guid);
 };
 const resetVerifiedUserIdentity = () => {
-    errorMgs.value = ''
+    errorMgs.value = '';
     verifiedUserIdentity.value = null;
     if (props.domain == UserType.I) emit('setVerifyResult', false);
```

Comment:
Do we only reset this input to "false" when it is IDIR user?
I think I can search BCeID user with "LOAD-3-TEST" verified and then change mind to input "LOAD-2-TEST" and still can submit.

Reply:
Hi Ian, it's got fixed